portal-impl/src/com/liferay/portal/service/persistence/impl/OrganizationFinderImpl.java

```diff
@@ -576,6 +591,94 @@ protected int countByOrganizationId(
 		return 0;
 	}
 
+	protected List<Object> doFindO_U_ByC_P(
+		long companyId, long parentOrganizationId,
+		QueryDefinition<?> queryDefinition) {
+
+		Session session = null;
+
+		try {
+			session = openSession();
+
+			StringBundler sb = new StringBundler(5);
+
+			sb.append(StringPool.OPEN_PARENTHESIS);
+
+			String sql = CustomSQLUtil.get(FIND_BY_C_P);
+
+			sql = StringUtil.replace(
+				sql, "SELECT organizationId",
+				"SELECT organizationId, 0 AS userId, name");
+			sql = StringUtil.replace(
+				sql, "(organizationId > ?) AND", StringPool.BLANK);
```

Comment:
I have found these SQL replacements in this commit confusing since the SQL query findByC_P as defined at this point in portal.xml do not contain "SELECT organizationId" or "(organizationId > ?) AND". I've noticed this code is changed in a later commit, but it would still be useful to fix this commit so that it's easier to make sense of it.